.github/workflows/build-cross.yml

```diff
@@ -51,7 +51,10 @@ jobs:
           ./configure --disable-warn-error --disable-ocamldoc \
               --disable-ocamltest --disable-stdlib-manpages \
               --disable-dependency-generation --prefix="$PREFIX" || res=$?
-          if ! [ "$res" = 0 ]; then cat config.log; exit "$res"; fi
+          if ((res)) ; then set +x
```

Comment:
I can never make my mind up about using `((`, because it looks like a "normal" logic expression, but of course exit code 0 ("false") is _good_!

If only to deal with my logical limitations, please could the variable be renamed from `res` to `failed` (as in `build-msvc.yml`) - it then means when I glance at it that it's clear that the log is only displayed _on error_!

Reply:
Done!